From 04f30148645a1fff9d409dcb84d52f97da0ecba1 Mon Sep 17 00:00:00 2001
From: Hyun Kwon <hyun.kwon@xilinx.com>
Date: Thu, 3 Sep 2020 12:39:36 -0700
Subject: [PATCH 1562/1851] v4l: xilinx: m2m: Remove an unused variable

commit 22b53c1f8a80113fde6c7660208afcb7e074894a from
https://github.com/Xilinx/linux-xlnx.git

drivers/media/platform/xilinx/xilinx-m2m.c:467:24: warning: variable 'dma' set but not used [-Wunused-but-set-variable]
  467 |   struct xvip_m2m_dma *dma;
      |                        ^

Signed-off-by: Hyun Kwon <hyun.kwon@xilinx.com>
Reviewed-by: Vishal Sagar <vishal.sagar@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-m2m.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/media/platform/xilinx/xilinx-m2m.c b/drivers/media/platform/xilinx/xilinx-m2m.c
index cb51eaa6e79b..97f6db39f881 100644
--- a/drivers/media/platform/xilinx/xilinx-m2m.c
+++ b/drivers/media/platform/xilinx/xilinx-m2m.c
@@ -464,13 +464,9 @@ static int xvip_pipeline_validate(struct xvip_pipeline *pipe,
 	media_graph_walk_start(&graph, entity);
 
 	while ((entity = media_graph_walk_next(&graph))) {
-		struct xvip_m2m_dma *dma;
-
 		if (entity->function != MEDIA_ENT_F_IO_V4L)
 			continue;
 
-		dma = to_xvip_dma(media_entity_to_video_device(entity));
-
 		num_outputs++;
 		num_inputs++;
 	}
-- 
2.31.1

